docs/cli.md

```diff
@@ -9,7 +9,7 @@ We plan to continue adding new features, such as interaction with MCP servers, a
 
 ## Usage
```

Comment:
The introductory section of the page should already mention that it has a CLI, as well as a web UI